### ISSUE TITLE
Keyboard navigation

### ISSUE THREAD

**Issue by ariejan** [Panonim/dynacat#63]
## The "problem"

Lack of keyboard navigation. I.e. switching to another page requires a mouse click (or relying on inefficient browser keyboard navigation). 

Looking at this from some distance, I can see more features related to this, like key-combo's per app/link, navigation, etc. 

## Configuration proposal

```yaml
key-bindings:
  next-page: tab
  previous-page: shift-tab

pages:
  # ...
  sites:
    - title: Pi-hole
      url: http://pihole.lan/admin
      key-binding: ctrl-p
```

**Comment by Panonim** [comment]
Added! You can try it with: `panonim/dynacat:beta-8cb0315`

**Comment by ariejan** [comment]
@Panonim I see the `data-key-bind` attributes for my pages (i.e. `data-key-bind="d b"`). But typing d, then b, does nothing. No messages in browser console either.

**Comment by Panonim** [comment]
Are you using the image provided by me or are you on `2.2.2`?

**Comment by Panonim** [comment]
Also you need to specify it yourself https://dynacat.artur.zone/beta/#configuration/key-bind
